docs/config.md

```diff
@@ -51,11 +51,12 @@ Supported keys include:
 
 | Key | Description | Supported Values | Default |
 | :--- | :--- | :--- | :--- |
-| `artifacts.taskrun.format` | The format to store `TaskRun` payloads in. | `in-toto`, `slsa/v1`| `in-toto` |
+| `artifacts.taskrun.format` | The format to store `TaskRun` payloads in. | `in-toto`, `slsa/v1`, `slsa/v2alpha1`| `in-toto` |
```

Comment:
I'd go a step further and remove it from the top level config docs for now and just leave it documented in slsa/README.md.

Reply:
Sounds good!